### PR TITLE
real color

### PR DESCRIPTION
**Pull request by bluelovers**
colors

```javascript
let line1 = '';
let line2 = '';

for (let i = 1 ; i <= 7 ; i++)
{
	line1 += `\u001b[3${i}m 3${i} \u001b[39m`;
	line2 += `\u001b[9${i}m 9${i} \u001b[39m`;

	//console.log(`\u001b[3${i}m 3${i} \u001b[39m`);
	//console.log(`\u001b[9${i}m 9${i} \u001b[39m`);
}

console.log(line1);
console.log(line2);
```

### CODE REVIEW

**Comment by Qix-** [comment]
What is this supposed to be doing?

**Comment by bluelovers** [comment]
@Qix-

**Comment by Qix-** [comment]
Im still not understanding what is being fixed here. Consoles have different colors, all ansi-styles does is use the corresponding codes.

**Comment by bluelovers** [comment]
maybe this like as https://github.com/chalk/ansi-styles/issues/26

just add colors that default support at console on mingw bash, windows cmd, powershell

**Comment by Qix-** [comment]
Oh, okay. Can you change `real` to `bright`? :)

**Comment by bluelovers** [comment]
rename to bright

File: index.js [hunk 1]
```diff
@@ -84,6 +84,11 @@ function assembleStyles() {
 				open: `\u001B[${style[0]}m`,
 				close: `\u001B[${style[1]}m`
 			};
+
+			Object.defineProperty(styles[styleName], 'data', {
+				value: style,
+				enumerable: false
+			});
```
Comment (Qix-) [inline index.js]: What does this do?

Comment (bluelovers) [inline index.js]: oh sry, that is anthor patch when wanna use raw data, without parse style.open, style.close back to color code

Comment (bluelovers) [inline index.js]: i can rollback if need

Comment (Qix-) [inline index.js]: yeah revert that and I'll merge the rest :)

Comment (bluelovers) [inline index.js]: @Qix- done

**Comment by Qix-** [comment]
Thanks :)

**Comment by sindresorhus** [comment]
@Qix- This is missing docs.

**Comment by Qix-** [comment]
Released as `3.1.0`. Thanks again!

**Pull request merged**